src/components/List/List.tsx

```diff
@@ -26,7 +26,7 @@ const asListItem = (child : React.ReactChild) =>
     : child;
 
 /** Ensure all children are <li> elements. */
-const asListItems = (children?: React.ReactNode) => 
+const asListItems = (children?: any) => 
```

Comment:
Why was this restriction relaxed and doesn't use `React.ReactChild`?

Reply:
`React.ReactNode` yielded a compilation error due to weirdness with `React.Children.map` overloads. This is an internal function so I let it go.

Reply:
I'll take another swing at it.